public/data/ayushlohmod.json

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
There's a missing `{` here.
```suggestion
{
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix